projects/lib/src/admin/components/page-editor/page-editor.component.ts

```diff
@@ -15,6 +15,7 @@ import { NgbModalRef, NgbModal } from '@ng-bootstrap/ng-bootstrap';
 import { PAGE_SCHEMA } from '../../constants/page-schema.constants';
 import { RequiredDeep } from '@ordercloud/headstart-sdk/dist/models/RequiredDeep';
 import { ResourceType } from '../../../shared/models/resource-type.interface';
+import { ParentResourceType } from '../../../shared/models/parent-resource-type.interface';
```

Comment:
Instead of this new interface, can we use the existing one for ResourceType? I don't think there will be a difference in the acceptable 'types' of both parameters .

Reply:
I can check with Oliver, but I believe that there is a difference.  I did try that, and I was getting an error saying 'type '... all of the `ResourceType` cannot be converted to type `'Catalogs' | 'Buyers' | 'Suppliers'`.  I believe that for ParentResourceType, only those three types are available.

Reply:
Oh okay that would make sense

Reply:
Ok - Oliver confirmed that the two types are different, and both necessary.